src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabsArr, onTabSelected, selected }) => {
+  const selectedTab = tabsArr.find(t => t.id === selected) || tabsArr[0];
+
+  const selectTabHadler = (nId) => {
+    const selTab = tabsArr.find(t => t.id === nId);
+
+    onTabSelected(selTab || tabsArr[0]);
```

Comment:
Consider adding a logic for checking ID here: (selTab.id !== selectedTab.id)

Reply:
done